docs/DataProviders.md

```diff
@@ -1,14 +1,60 @@
 ---
 layout: default
-title: "Using Data Providers"
+title: "Data Fetching"
 ---
 
-# Setting Up The Data Provider
+# Data Fetching
+
+You can build a react-admin app on top of any API, whether it uses REST, GraphQL, RPC, or even SOAP, regardless of the dialect it uses. This works because react-admin doesn't user `fetch` directly. Instead, it uses a Data Provider object to interface with your API, and [react-query](https://tanstack.com/query/v3/docs/react/overview) to handle data fetching.
+
+## The Data Provider
+
+Whenever react-admin needs to communicate with your APIs, it does it through an object called the `dataProvider`. The `dataProvider` exposes a predefined interface that allows react-admin to query any API in a normalized way. 
+
+<img src="./img/data-provider.png" class="no-shadow" alt="Backend agnostic" />
+
+For instance, to query the API for a single record, react-admin calls `dataProvider.getOne()`: 
+
+```js
+dataProvider
+    .getOne('posts', { id: 123 })
+    .then(response => {
+        console.log(response.data); // { id: 123, title: "hello, world" }
+    });
+```
+
+It's the Data Provider's job to turn these method calls into HTTP requests, and transform the HTTP responses to the data format expected by react-admin. In technical terms, a Data Provider is an *adapter* for an API. 
+
+Thanks to this adapter system, react-admin can communicate with any API, whether it uses REST, GraphQL, RPC, or even SOAP, regardless of the dialect it uses. Check out the [list of supported backends](./DataProviderList.md) to pick an open-source package for your API.
```

Comment:
This paragraph is almost a repetition of the first paragraph of the introduction

```suggestion
Thanks to this adapter system, react-admin can communicate with any API. Check out the [list of supported backends](./DataProviderList.md) to pick an open-source package for your API.
```